ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.13-nightly-x86.yaml

```
@@ -114,6 +114,17 @@ tests:
     - ref: openshift-qe-perfscale-aws-data-path-sg
     - chain: openshift-qe-data-path-tests
   timeout: 5h0m0s
+- always_run: false
+  as: control-plane-3nodes
+  steps:
+    cluster_profile: aws-perfscale-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "3"
+    test:
+    - chain: openshift-qe-control-plane
```

```suggestion
    - chain: create-infra-move-ingress-monitoring-registry
    - chain: openshift-qe-control-plane
```
@svetsa-rh I think we can include the chain of 'create-infra-move-ingress-monitoring-registry' here, before 'create-infra-move-ingress-monitoring-registry', so that it can be run after the install 'create-infra-move-ingress-monitoring-registry' and before our real test 'create-infra-move-ingress-monitoring-registry'.

---

Hi Qiujie, that will be very similar to using existing `ipi-aws` workflow and adding chains for further setup. We will also have to mention that for every test. IMO, we should just use `workflow: openshift-qe-installer-aws` (which we already have in the repo from a long time).

For adding support for new profiles, if you want, you can try that approach i.e., use `workflow: openshift-qe-installer-aws` and add any additional pre-requisites as chains before running tests.

---

Just to clarify, I am more inclined to creating new workflows for new profiles as discussed here https://redhat-internal.slack.com/archives/C05CDC19ZKJ/p1699380498538189 
and use `workflow: openshift-qe-installer-aws` for basic AWS setup for Perfscale.